drivers/hmis/app/models/hmis/ce/client_match/candidate.rb

```diff
@@ -0,0 +1,41 @@
+# an eligible, prioritized client for a given policy
+module Hmis::Ce::ClientMatch
```

Comment:
One take away I have from CAS where we called this ClientOpportunityMatch, is that, we just call them matches.  Can we rename this module to `Match` or is that problematic?  I doubt we'll ever match anything other than a client.